db/migrations/20220609044825-move-github-app-id-to-subscriptions.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
```

Comment:
Moved gitHubAppId from Installations to Subscriptions to align with our data model update.